src/plays/tip-calculator/styles.css

```diff
@@ -0,0 +1,4 @@
+/* enter stlyes here */
+:root {
```

Comment:
@ayushsgithub Let's not override the :root selector. Could you please add a style in your play level instead? Rest of the changes look good?

Reply:
Hey @atapas sir, removed the root selector as per the required changes